CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## UNRELEASED
 * Feature - Support for provisioning Tasks with ENIs
+* Feature - Support pulling from ecr with specified IAM role in task definition
```

Comment:
nit: `ecr` -> `ECR`, `role` -> `Role` and `task definition` -> `Task Definition`

Reply:
I think guidance from @nrdlngr would be for role, task, and task definition to remain lower-case.

Reply:
Yes. Role, task, and task definition would use sentence case (Only capitalize if it's the first word in a sentence), and all AWS service abbreviations should be capitalized. I made some comments to this effect in my review, but I might not have caught them all.